bundle-size/test/app.test.js

```diff
@@ -57,7 +57,7 @@ async function waitUntilNockScopeIsDone(nocks) {
   nocks.done();
 }
 
-describe('bundle-size', async () => {
+describe('bundle-size', () => {
```

Comment:
Curious: If you are removing the `async` keyword, shouldn't you also be making the test synchronous by say, returning a promise? (Assuming the tests were asynchronous to begin with.)

Reply:
The tests themselves do return a promise, it's the test-suite (`describe('...', functionThatDefinesTestSuite)` that should not be